go/libraries/doltcore/sqle/enginetest/dolt_queries_merge.go

```diff
@@ -5201,6 +5240,7 @@ var ThreeWayMergeWithSchemaChangeTestScripts = []MergeScriptTest{
 		Assertions: []queries.ScriptTestAssertion{
 			{
 				Query:    "call dolt_merge('right');",
+				SkipResultsCheck: true,
```

Comment:
Is there a way to wildcard this? Seems sad to lose all the asserts.

Reply:
Right? I was thinking about this the other day... had a hacky idea where we could have a special/magic string like `"COMMIT HASH"` or `"SKIP VALIDATION"` and our test framework would skip validating that field. I'm not crazy about that idea, but it would be really nice to still assert the other values being returned are what we expect.

Reply:
Actually... I'm warming up to this, especially if we have the magic string tucked into a const and could just reference `SKIP_VALIDATION` instead of repeating the magic string. Given how many tests this change makes us needs to skip, I think an easy approach like this would be worth doing now.